### ISSUE TITLE
Add master tone control to the mixer

### ISSUE THREAD

**Issue by johnnovak** [dosbox-staging/dosbox-staging#2050]
Add home stereo style tone controls to the master audio channel to allow users fine-tune the general sound signature of the summed audio output.

Certain 90s audio cards had a similar feature in the form of master bass and treble controls (e.g. AWE32/64, and the AdLib Gold). This feature should mimic that by adding an adjustable low-shelf and high-shelf filters with fixed cutoff frequencies before the master high-pass filter. 

The bass and treble levels could be set from -12 to +12 dB, and both shelf filters should be of the Baxandall variety which is found in the tone control section of virtually all home stereos and hi-fi amplifiers. Basically, the bass and treble implementation of AdLib Gold could be copied as-is, maybe with a few minor tweaks to the cutoff frequencies.

The tone controls could be configured as follows:

```ini
[mixer]
bass = 3
treble = -1.5
```

**Comment by Burrito78** [comment]
German speaker manufacturer Nubert Electronik did design a new tone control for his speakers, see below. They call it "Klangwaage".

![atm-kompaktbox-bassfrequenzgang_1920x1920](https://github.com/dosbox-staging/dosbox-staging/assets/61995128/7dcfaec0-0a1b-454f-8a48-9a1bd2147501)

![atm-kompaktbox-mid-high-frequenzgang_1920x1920](https://github.com/dosbox-staging/dosbox-staging/assets/61995128/3fe0fc25-ff0f-4786-931c-0a2fa932c917)

Source: https://www.nubert.de/atm-kompakt/a038386

Just an idea, needs listening tests for sure.

**Comment by johnnovak** [comment]
Yeah not a bad idea, it looks like a regular Baxandall low shelf combined with a tilt filter as the mid/high filter. Problem is the IIR library we use doesn't have a tilt filter and I'm not quite a DSP expert... but we'll see 😄 My original "standard" low + high shelf filter idea is straightforward and easy to implement.
